frame/composable-traits/src/mosaic.rs

```diff
@@ -0,0 +1,107 @@
+//! Traits used in the implementation of the Mosaic pallet.
+
+use frame_support::dispatch::DispatchResultWithPostInfo;
+
+/// Trait containing the business logic relevant to managing the Relayer of the Mosaic pallet.
+pub trait RelayManager {
```

Comment:
Or just Relayer

Reply:
How about `RelayerInterface`? `Relayer` is already a type alias in this scope. Considering this acts as an interface for the Relayer, this might also be more informative. @KaiserKarel